src/Runner/Runner.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {EventEmitter} from 'events';
+import * as vscode from 'vscode';
+
+import {Balloon} from '../Utils/Balloon';
+import * as helpers from '../Utils/Helpers';
+import {Logger} from '../Utils/Logger';
+
+const K_MODEL_SELECT: string = 'model-select'
+
+export type modelExt = 'pb'|'tflite'|'onnx';
+
+export class Runner extends EventEmitter implements helpers.FileSelector {
```

Comment:
IHMO, this could be divided to two roles?
1. sth to load model: load models from an event
1. runner: just runs a model loaded